### PR TITLE
Added password_validator

### PR DESCRIPTION
**Pull request by makarman**
## Code reviewers

- [x] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/tests/unittests/validators/testvalidators.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+"""Validators Test."""
```
Comment (redmyt) [inline eventually/tests/unittests/validators/testvalidators.py]: Please emphasize the name of the module with `====...` chars. It makes it more readable with sphinx docs =)

File: eventually/utils/validators.py [hunk 2]
```diff
@@ -120,3 +121,21 @@ def list_of_int_validator(value):
     if not all(isinstance(item, int) for item in value):
         return False
     return True
+
+
+def password_validator(password):
+    """
+    Function that provides passwor validation.
```
Comment (redmyt) [inline eventually/utils/validators.py]: password not passwor

File: eventually/utils/validators.py [hunk 3]
```diff
@@ -120,3 +121,21 @@ def list_of_int_validator(value):
...
+    :param password: the password itself.
+    :type password: string
+
+    :return: `True` if value if valid and `False` if it is not.
+    """
+    if string_validator(password, min_length=4, max_length=32):
+        template = re.compile(r'^(?=.*?\d)(?=.*?[A-Z])(?=.*?[a-z])[A-Za-z\d]{4,32}$')
+        if template.match(password):
+            return True
+    return False
```
Comment (redmyt) [inline eventually/utils/validators.py]: This `if` statement is not necessary because if we return nothing from the function the result will be `none`.

Comment (makarman) [reply eventually/utils/validators.py]: Okey)

File: eventually/utils/validators.py (hunk 2 above)
Comment (makarman) [reply eventually/utils/validators.py]: Thanks I missed

Comment (makarman) [reply eventually/utils/validators.py]: Done

File: eventually/tests/unittests/validators/testvalidators.py (hunk 1 above)
Comment (makarman) [reply eventually/tests/unittests/validators/testvalidators.py]: Done

**Review by redmyt — DISMISSED** [review]

File: eventually/utils/validators.py [hunk 4]
```diff
@@ -120,3 +121,20 @@ def list_of_int_validator(value):
     if not all(isinstance(item, int) for item in value):
         return False
     return True
+
+
+def password_validator(password):
+    """
+    Function that provides password validation.
+
+    Pasword should consist of: uppercase letters: A-Z, lowercase letters: a-z, numbers: 0-9
```
Comment (redmyt) [inline eventually/utils/validators.py]: forget one `s` =)
pasSword not pasword

Comment (makarman) [reply eventually/utils/validators.py]: one second)

**Review by redmyt — DISMISSED** [review]

**Review by ssOleg — APPROVED** [review]

**Pull request merged**